sqsdomain/cw_pool.go

```diff
@@ -42,7 +51,15 @@ func NewCWPoolModel(contract string, version string, data CWPoolData) *CosmWasmP
 }
 
 func (model *CosmWasmPoolModel) IsAlloyTransmuter() bool {
-	return model.ContractInfo.Matches("crates.io:transmuter", "3.0.0")
+	name := "crates.io:transmuter"
+	version := ">= 3.0.0"
+
+	constraints, err := semver.NewConstraint(version)
+	// this must never panic
+	if err != nil {
+		panic(err)
+	}
+	return model.ContractInfo.Matches(name, constraints)
```

Comment:
I like this approach. It is now extensible without having to make code changes to both, the `osmosis` node side and `sqs` for minor migrations.

Can we also add constraint of semver being less than <4.0.0 so that, if there are any future versions, those are auto-accounted for without more code changes?

Reply:
At least for now that I'm aware of, 4.0.0 will still be compatible, so I would avoid constraining that if you don't want code change. Until there is a version the requires changes in logic, I think leave this lax constraints would be easier to operate.

Reply:
I think we should have a standard where a separate major means a breaking change.

If `4.0.0` is compatible, can it be `3.1.0` instead?

Reply:
It's breaking other parts. For 4.0.0 we are planning to remove some entrypoints, putting more constraints on denom, but just does not effect sqs.

Reply:
Got it, let's leave it as is. Thank you!